exercises/spiral-matrix/spiral_matrix_test.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from spiral_matrix import spiral
+
+
```

Comment:
Please add a comment referencing the current version of the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/spiral-matrix/canonical-data.json). For the format of this comment, please see #784.

Reply:
Done. Thanks for the hint.